org.goldcoinproject.goldcoin-qt.metainfo.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.goldcoinproject.goldcoin-qt</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <name>Goldcoin</name>
+  <summary>Discover the Gold Standard of Digital Currency.</summary>
+  <description>
+    <p>
+      Goldcoin is the first Digital Gold. A super secure value store and cash payment system. Fully decentralized and distributed. Fast and Free. It's the Money for a New Generation. Nakamoto used the word Gold multiple times in his Bitcoin whitepaper to describe the inner workings of his original protocol. We've preserved it, improved it, and made it stronger and faster. Goldcoin (GLC) represents economic freedom to 8 billion people Worldwide.
+    </p>
+  </description>
+  <launchable type="desktop-id">org.goldcoinproject.goldcoin-qt.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/expatjedi/flatpak-data/main/wallets_splash.png</image>
+    </screenshot>
+  </screenshots>
+  <url type="bugtracker">https://github.com/expatjedi/org.goldcoin.goldcoin-qt/issues/new?title=[flatpak]</url>
+  <url type="homepage">https://www.goldcoinproject.org/</url>
+  <developer_name>Goldcoin Developers</developer_name>
+  <update_contact>kbicer@proton.me</update_contact>
+  <content_rating type="oars-1.1" />
```

Comment:
IMHO there should be something about "money" in the content rating.

Reply:
Our wallet does not have an in-app activity of producing advertisements for commercial products or services so closing this subject.